utils.py

```diff
@@ -43,7 +47,74 @@ def server_response_decode(message):
             remainder = message_parts[1][b_string_len + 2 :]
 
     elif message[0] == ":":  # responses should start with '(integer)'
-        print("integer")
-        value = int(message_parts[0])
+        value = f"(integer) {message_parts[0]}"
         remainder = message_parts[1]
     return value, remainder
+
+
+def generate_redis_formatted_array(key_or_value, cli_message):
+    # Generates start of array taking length of list based on number of arguments passed to it.
+    command_length = len(cli_message)
+    total_length = 1 + len(key_or_value)
+    encoded_message_start = f"{ARRAY_START}{total_length}{SEPARATOR}{BULK_STR_START}{command_length}{SEPARATOR}{cli_message}{SEPARATOR}"
+
+    # Creates and returns redis formatted array to send to Redis server
+    for key in key_or_value:
+        encoded_message_start += (
+            f"{BULK_STR_START}{len(key)}{SEPARATOR}{key}{SEPARATOR}"
+        )
+    return encoded_message_start
+
+
+def command_handler(message, dict):
```

Comment:
it looks like this is only used by the server and should live there instead

Reply:
Relocation has commenced! 🐧